server/src/main/scala/services/graphstages/ArrivalsGraphStage.scala

```diff
@@ -273,34 +273,36 @@ class ArrivalsGraphStage(name: String = "",
     }
 
     def mergeArrival(key: UniqueArrival): Option[Arrival] = {
-      val maybeArrival = liveBaseArrivals.get(key)
+      val maybeArrival = ciriumArrivals.get(key)
       val maybeSanitisedLiveBaseArrival = maybeArrival.map(arrivalDataSanitiser.withSaneEstimates)
 
-      val maybeBestArrival: Option[Arrival] = (liveArrivals.get(key), maybeSanitisedLiveBaseArrival, forecastBaseArrivals.get(key)) match {
-        case (Some(liveArrival), Some(baseLiveArrival), _) =>
-          val mergedLiveArrival = LiveArrivalsUtil.mergePortFeedWithLiveBase(liveArrival, baseLiveArrival)
-          val sanitisedLiveArrival = ArrivalDataSanitiser
-            .arrivalDataSanitiserWithoutThresholds
-            .withSaneEstimates(mergedLiveArrival)
-          Option(sanitisedLiveArrival)
+      val maybeBestArrival: Option[Arrival] = (liveArrivals.get(key), maybeSanitisedLiveBaseArrival, aclArrivals.get(key)) match {
+        case (Some(liveArrival), Some(ciriumArrival), _) =>
+          if (ciriumArrival.Origin == liveArrival.Origin) {
```

Comment:
are we sure this is only going to affect cirium arrivals?

Reply:
Yes the change is specifically around the cirium arrivals only